docs/products/kafka/howto/schema-registry.md

```diff
@@ -66,77 +81,83 @@ use a **click record** schema defined in JSON and stored in a file named
 }
 ```
 
-The JSON configuration above defines a schema named `ClickRecord` in the
-namespace `io.aiven.avro.example` with fields `session_id`, `browser`,
-`campaign`, `channel`, `referrer` and `ip` and related data types.
+This schema defines a record named `ClickRecord` in the namespace `io.aiven.avro.example`
+with fields `session_id`, `browser`, `campaign`, `channel`, `referrer` and `ip` and
+related data types.
 
 Once the schema is defined, compile it either **manually** or **automatically**.
 
+To generate Java classes from Avro, Protobuf, or JSON Schema files, see
+[Generate Java classes from schemas](/docs/products/kafka/howto/generate-java-classes-from-schemas).
+
 ### Manual schema compilation
 
-In case of manual schema compilation, download `avro-tools-1.11.0.jar`
-from [https://avro.apache.org/releases.html](https://avro.apache.org/releases.html) or via maven using the
-following:
+To compile manually, download`avro-tools-1.11.0.jar` from
```

Comment:
```suggestion
To compile manually, download`avro-tools-1.12.0.jar` from
```

This is going to be fun to maintain!  1.12.1 should be coming out in the next few months :laughing: